ospfd/ospf_te.c

```diff
@@ -71,6 +77,7 @@ struct ospf_mpls_te OspfMplsTE;
 
```

Comment:
Well, this is historic. The original code has more than 20 years, and used a static variable.

In addition, there is not too much reason to add MPLS TE context to osfp os ospf_area structure. Indeed, this will render the code more complex when ospf or ospf_area context are not available.

Reply:
ok